src/main/java/mafia/mafiatogether/job/ui/JobV2Controller.java

```diff
@@ -0,0 +1,36 @@
+package mafia.mafiatogether.job.ui;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import mafia.mafiatogether.job.application.JobService;
+import mafia.mafiatogether.job.application.dto.request.JobExecuteAbilityRequest;
+import mafia.mafiatogether.job.application.dto.response.JobExecuteAbilityResponse;
+import org.springframework.data.redis.core.StringRedisTemplate;
+import org.springframework.messaging.handler.annotation.DestinationVariable;
+import org.springframework.messaging.handler.annotation.MessageMapping;
+import org.springframework.messaging.handler.annotation.Payload;
+import org.springframework.stereotype.Controller;
+
+@Slf4j
+@Controller
+@RequiredArgsConstructor
+public class JobV2Controller {
+
+    private final StringRedisTemplate stringRedisTemplate;
+    private final JobService jobService;
+    private final ObjectMapper objectMapper;
+
+    @MessageMapping("/jab/skill/{code}/{name}")
```

Comment:
```suggestion
    @MessageMapping("/job/skill/{code}/{name}")
```
아니면 레거시 코드의 url이랑 혼동 충돌되서 이렇게 한건가?
차라리 `"/v2/job/skill/{code}/{name}"`으로 하고 추후 레거시 지울때 바꾸는건 어때?

Reply:
이거 오타..